application/src/main/java/org/opentripplanner/updater/vehicle_rental/datasources/GbfsStationStatusMapper.java

```diff
@@ -83,13 +73,52 @@ VehicleRentalStation mapStationStatus(VehicleRentalStation station) {
       .withSpacesAvailable(spacesAvailable)
       .withVehicleSpacesAvailable(vehicleSpacesAvailable)
       .withSpacesDisabled(spacesDisabled)
-      .withIsInstalled(isInstalled)
       .withIsRenting(isRenting)
       .withIsReturning(isReturning)
-      .withLastReported(lastReported)
+      .withReturnPolicy(returnPolicy(status))
       .build();
   }
 
+  /**
+   * Extract the available parking places from a variety of potentially incomplete information.
+   */
+  private Map<RentalVehicleType, Integer> vehicleSpaces(
+    VehicleRentalStation station,
+    GBFSStation status,
+    int spacesAvailable
+  ) {
+    var typesAvailable = status.getVehicleTypesAvailable();
+    var docksAvailable = status.getVehicleDocksAvailable();
+    if (docksAvailable == null && typesAvailable == null) {
```

Comment:
I'm assuming the bicycle fallback is here for backwards compatibility? But is that really needed now that the routing can fall back to the ANY_TYPE?